Enterprise-CMCS/macpro-mdct-mcr
# Add MCPAR dashboard Playwright tests

### Description

- Add tests for MCPAR dashboard - `/mcpar`
- DRY up base fixture, utils imports, auth context

### Related ticket(s)


CMDCT-4133

---
### How to test


1. Make sure your .env is up-to-date - `./run update-env`
2. Run `yarn test:e2e` or `yarn test:e2e-ui`
3. All tests should pass





---
### Pre-review checklist
